src/blocks/EventAccordion/Layout/index.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+import Collapsed from "./Collapsed";
+import Open from "./Open";
+
+const Layout = styled.div`
```

Comment:
Can you please name your components more semantically using the file path? E.g. `EventAccordionLayout`